packages/edit-site/src/components/global-styles/preview-highlighted-colors.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { __unstableMotion as motion } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useStylesPreviewColors } from './hooks';
+
+export default function PreviewHighlightedColors( {
```

Comment:
This component renders a single frame for a preview I take it.

`<StylesPreviewColors />` and `<StylesPreview />` both use the `<PreviewWrapper />`, so they're stand alone. 

I think all the "preview" file names had me scratching my beard for a moment. 

It's not a huge deal, since it's not a public API, but baybe we could restrict the word "previews" to components that render a "preview" to the page as it's currently understood in trunk?

This small component could always be exported from packages/edit-site/src/components/global-styles/preview-colors.js

This is an observation only and optional 👍🏻

Reply:
Done in fa60cef4be